### PR TITLE
Push a whole chunk of my project

### PR DESCRIPTION
**Pull request by Rifat8080**
- Added linters config
- Added webpack
-

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+<body>
+  <section class="todo-container">
+    <div class="todo-list todo-intro"><h4 class="todo-intro-title">Demo</h4><i class="fa-solid fa-rotate"></i></div>
+    <div class="todo-list todo-add">
+      <input type="text" placeholder="Add to your list">
+      <i class="fa-solid fa-arrow-left-long"></i>
+    </div>
+    <ul class="todo-group"></ul>
+
+      <button type="button" class="clear-todo"><span>Clear all completed</span></button>
```
Comment (batoolfatima2135) [inline docs/index.html:19]: As recommended by HTML best practice, it would be better if there are no empty lines without any reason

**Review by batoolfatima2135 — CHANGES_REQUESTED** [review]
Hi @Rifat8080 ,
You have done a great job so far! 👍 

### STATUS: CHANGES REQUIRED ♻️ ♻️
There are some issues that you still need to work on to go to the next project but you are almost there!

### Highlights

- All linter checks are passing ✔️ ✔️
- Your PR is professional 👍 👍
- Professional GitHub flow 👍 👍

### Required Changes 
Check the comments under the review.

Keep up the great work! 👏  Happy Coding!!  😊

File: docs/index.html (hunk 1 above)
Comment (Rifat8080) [reply docs/index.html:19]: Thank you for letting me know the changes I need to make to improve my project 👍

**Review by batoolfatima2135 — APPROVED** [review]
Hi, @Rifat8080,
Thank you so much for considering my suggestion. 😊 Your project is complete! There is nothing else to say other than... it's time to merge it. ✨
Congratulations! 🎉🎊

### Highlights 🔥

- Linters well setup and passing ✔️
- Good use of GitHub flow ✔️
- All functionalities work as expected ✔️

Cheers, and Happy coding!👏👏👏

**Pull request merged**